README.md

```diff
@@ -24,23 +24,23 @@ development practices, and clean project structure.
 
 ### Metarhia and Impress application server way
 
-- Applied code need to be simple and secure, so we use code sandboxing with v8
+- Applied code needs to be simple and secure, so we use code sandboxing with v8
   isolated contexts, worker threads and minimal trusted npm dependencies;
-- Domain code need to be separated from system code; so we use DDE, layered
-  (onion) architecture, DI, SOLID:DIP principle and contract-based approach;
+- Domain code needs to be separated from system code; so we use DDE, layered
+  (onion) architecture, DI, SOLID, DIP principle and contract-based approach;
```

Comment:
```suggestion
  (onion) architecture, DI, SOLID:DIP principle and contract-based approach;
```

Reply:
Added suggested changes.
Sorry, didn't get it the first time. It's DIP from SOLID only, not "SOLID and DIP"